.github/workflows/bench.yml

```diff
@@ -43,12 +43,16 @@ jobs:
 
       - name: Truncate benchmark git history
```

Comment:
Note that this doesn't remove the historical benchmark data, just the benchmarking commits that added them. The full history is preserved as dated entries within the file, so we don't need the commits. As well as recording the date, the entries also record which commit on `master` was used for the code that was benchmarked.